ground/src/main/java/com/google/android/ground/ui/datacollection/DropAPinTaskFragment.kt

```diff
@@ -30,8 +30,9 @@ import javax.inject.Inject
 
 @AndroidEntryPoint
 class DropAPinTaskFragment : AbstractMapContainerFragment(), TaskFragment<DropAPinTaskViewModel> {
-
   override lateinit var viewModel: DropAPinTaskViewModel
+  // Not needed for DropAPinTaskFragment
+  override var position = -1
```

Comment:
It's probably overkill, but we could also implement this out as a concretization over the TaskFragment, e.g. some PositionlessTaskFragment<T>. Probably not worth it for now, but if we wind up with 3+ classes like this I'd consider it.

Reply:
Yea I think that sounds good, if we end up adding many more that don't need a reference to the position I think we could revisit at that point.